README.md

```diff
@@ -1,70 +1,162 @@
+First, run the appropriate setup script for your system. This will install relevant packages, walk through Android Studio configuration, etc.
 
-## Running
+**Note:** You will still need to [configure an Android Virtual Device (AVD)](https://developer.android.com/studio/run/managing-avds#createavd) after running the script.
 
-```yarn``` must be installed for this project, as it does a better job installing dependendies across platforms.  The dev_setup script above should have done that for you.
+#### Linux/MacOS
 
-### Install modules to the correct locations
+```
+dev_setup.sh
+```
+
+#### Windows
+
+```
+dev_setup.bat
+```
 
-1. ```yarn install``` (always do this after a new clone or checkout)
-2. ```yarn install:pod``` (additional step for macOS if you want to do an iOS build)
+## Running
+
+**Note:** In some cases, these procedures can lead to the error `Failed to load bundle - Could not connect to development server`. In these cases, kill all other react-native processes and try it again.
 
-### Run the app in a simulator
+#### Android (Windows, Linux, macOS)
 
-To run in the Android simulator (Windows, Linux, macOS):
 ```
 npx react-native run-android
 ```
-or (on macOS only)
+
+Device storage can be cleared by long-pressing on the app icon in the simulator, clicking "App info", then "Storage", and lastly, "Clear Storage".
+
+#### iOS (macOS only)
+
 ```
+yarn install:pod ## only needs to be ran once
 npx react-native run-ios
 ```
 
-NOTE: In some cases, these procedures can lead to the error `Failed to load bundle - Could not connect to development server`. In these cases, kill all other react-native processes and try it again.
+Device storage can be cleared by clicking "Hardware" on the system toolbar, and then "Erase all content and settings".
 
-## Contributing
+### Release Builds
 
-Read the [contribution guidelines](CONTRIBUTING.md).
+Generating a release build is an optional step in the development process.
+
+- [Android instructions](https://reactnative.dev/docs/signed-apk-android)
+
+### Debugging
+
+[React Native ships with a debugging tool](https://reactnative.dev/docs/debugging#accessing-the-in-app-developer-menu) that allows you to inspect code, set debugging breakpoints, and more. Each platform has a different command to open the debugger.
+
+**Android:** `⌘D`
+
+**iOS:** `⌘M`
 
+## Testing
+
+Tests are ran automatically through Github actions - PRs are not able to be merged if there are tests that are failing.
+
+### Unit Test
+
+To run the unit tests:
+
+```
+yarn test --watch
+```
+
+To update the snapshots:
+
+```
+yarn update-snapshots
```

Comment:
For the new folks on the project, it may make sense to clarify what snapshots are and why you might want to update them

Reply:
This probably warrants being moved into a unit testing wiki page so that we can eventually remove snapshots as a thing.

Reply:
I'll just throw a link to the jest docs for the time being.